### ISSUE TITLE
Errors when search result returns nothing.

### ISSUE THREAD

**Issue by Enderlein** [vn-ki/anime-downloader#12]
When a search result in `anime watch --new` returns nothing (i.e. `anime watch --new "Kanpai"`), it still displays a list of search results, even though it's completely empty. 
![image](https://user-images.githubusercontent.com/31358215/41551298-66951872-72f9-11e8-8c6a-94916b614b0d.png)

If you select an option, it aborts the search, telling you to input a number less than 31:
![image](https://user-images.githubusercontent.com/31358215/41551342-81feb96a-72f9-11e8-895e-3d9cd7e56493.png)

I think this should be made more graceful. There should either be an option for the user to abort the search on their own or it should tell the user that there are no search results and then abort.

**Comment by vn-ki** [comment]
Yeah. There are a lot of things to be ironed out in that category. 

Funny of me to assume one will get 30 results for every search. If you're willing to open a PR, I'm happy to merge it.

**Comment by vn-ki** [comment]
If you are okay with it, let @KashTag take up this issue. I'm sure you'll find a lot more bugs!

**Comment by Enderlein** [comment]
No problem at all. I'll keep looking.

**Comment by vn-ki** [comment]
That's great @Enderlein.

@KashTag If you are interested, please look into this issue.

**Comment by KashTag** [comment]
Sorry I'm still new to Python so I took my stupid time working this out.
I changed the code like this. 

![capture](https://user-images.githubusercontent.com/39453935/41577751-4cabec0c-73ac-11e8-9381-5a78651965d8.PNG)

I wish I knew how to make it rerun the function instead of exiting.

Now it shows the following:

![capture2](https://user-images.githubusercontent.com/39453935/41577778-6547a0da-73ac-11e8-9758-a5c765f13f8d.PNG)
